content/services/user-support.mdx

```diff
@@ -5,25 +5,39 @@ description: |
   We strategically facilitate access to essential university research services.
 ---
 
-<ContentSection title="Advanced Research IT Solutions">
-
-The Advanced Research IT Solutions group offers specialized IT expertise to academic departments and centers engaged in research across the university. Our dedicated team of system engineers provides personalized service and advanced administration and troubleshooting for:
-
-- Linux and Windows systems
-- Web-based services
-- Research data management
-- Database support
-- Globus Data Transfer
-- OSCAR HPC (High-Performance Computing)
-- Robust file storage solutions
-- GitHub Enterprise
-- Kubernetes
-- Firebase Web App Hosting
-- Virtual Machine Hosting
-- Managed Database Hosting
-- Consultancy-based Research Software Engineering
-
-<Button href="mailto:ccv-res-sys@brown.edu">
-  Contact Us
-</Button>
+<ContentSection title="How We Support Brown">
+
+We're here to help you navigate our services and make your research computing as seamless as possible. We provide
+support for our High-Performance Computing cluster, [Oscar](/services/oscar), our secure compute and storage system,
+[Stronghold](/services/stronghold), various [storage and transfer services](/services/storage) like Globus and
+Hibernate, and more!
+
+We can assist with:
+
+- **Onboarding** to CCV systems.
+- **Troubleshooting** and debugging your code.
+- **Consulting** on which of our systems best fits your research needs.
 </ContentSection>
+
+<ContentSection title="How to Get Help">
+We offer a variety of support options designed to fit your needs. Choose the method that works best for you.
+
+### Documentation & Self-Service
+
+Our comprehensive [documentation](https://docs.ccv.brown.edu/documentation/) is the fastest way to find answers to common questions about using our systems.
+
+### Workshops & Tutorials
+
+- Join our **weekly online HPC tutorials** during the spring and fall semesters. Please check our our [events schedule](https://events.brown.edu/ccv/all) for upcoming sessions.
```

Comment:
same here:
`<a href="https://events.brown.edu/ccv/all" target="_blank" rel="noopener noreferrer">events schedule</a>`

Reply:
Should we link to events.brown or to our events section on the homepage? I guess I kind of like it going to the more comprehensive events.brown.edu in this case since the events section tends to focus on more of the near-term events. Thoughts?